src/components/EmptyArea.tsx

```diff
@@ -0,0 +1,45 @@
+import { Box, Center, VStack, Text, HTMLChakraProps } from "@chakra-ui/react"
+import { ReactFragment } from "react"
+
+import { EmptyBoxImage } from "assets"
+
+export interface EmptyAreaOption extends HTMLChakraProps<"div"> {
```

Comment:
nit: could we rename this to `EmptyAreaProps`? i think the convention is to use `xxxProps` so we should stick to it!

also, we should probably extend `StackProps` from chakra as the wrapper element is a `VStack`